dojo_plugin/pages/dojo.py

```diff
@@ -59,6 +59,49 @@ def view_dojo_path(dojo, path):
     else:
         abort(404)
 
+
```

Comment:
Could you add a test case for this? Only a matter of time until we break this endpoint otherwise :crying_cat_face:

Reply:
I'll take a look at the other tests and add a testcase 😄

Reply:
![grafik](https://github.com/user-attachments/assets/b2ac3577-af9e-4034-8d79-4e618a6ded01)
added